astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx

```diff
@@ -822,5 +822,6 @@ FusionAuth Cloud has the same [limitations](/docs/get-started/core-concepts/limi
 * If you are on FusionAuth Cloud and you find that some requests are failing, it is possible you are being rate limited. This isn't intentional, but an automated part of our infrastructure to ensure FusionAuth Cloud performance and security. If you are rate limited but need these requests to occur, please open a [support ticket](https://account.fusionauth.io/account/support/) with details.
 * If you want to run Advanced Threat Detection, an Enterprise feature, you'll need a cloud deployment with sufficient memory. Currently that means it must be a Large or X-Large size.
 * With HA and other multi-node deployments, requests are passed through a load balancer. When making requests to node specific metrics endpoints such as `/api/status` or `/api/prometheus/metrics` each request may return different results because the response is specific to the service node responding to the request.
-* If you are using the Elasticsearch search engine, you may not modify the Elasticsearch settings or view the Elasticsearch index directly. Among other things, this means that you can't use some of the [troubleshooting steps](/docs/operate/troubleshooting/troubleshooting) available to users self-hosting FusionAuth.
+* You may not modify the Elasticsearch settings or view the Elasticsearch index directly. Among other things, this means that you can't use some of the [troubleshooting steps](/docs/operate/troubleshooting/troubleshooting) available to users self-hosting FusionAuth.
 * OpenTelemetry data is not available on FusionAuth Cloud deployments.
+* There is a limit of 1000 `user.data` fields and 1000 `registration.data` fields.
```

Comment:
I'm pretty sure that this is not accurate. I believe that limit applies to the entire index, and not the individual data columns. Since users and registrations are indexed together, and this would include the other fields not in the data columns, the practical limit is much lower than this statement would suggest

see: https://www.elastic.co/guide/en/elasticsearch/reference/7.17/mapping-settings-limit.html

Reply:
Awesome, thanks, I'll update!